DEV-4759: list submissions start end dates (#1828)
fedspendingtransparency/data-act-broker-backend: dataactbroker/README.md

- `fabs`: (boolean) if the submissions listed should be FABS or DABS (True for FABS). Defaults to `False` if not provided.
- `filters`: (dict) additional filters to narrow the results returned by the endpoint. Possible filters are:
    - `submission_ids`: ([integer]) an array of integers or strings that limits the submission IDs returned to only the values listed in the array.

>     - `last_modified_range`: (dict) a start and end date for the last modified date range. If both are not provided, the search is unbounded in that direction (e.g. if the end date is not provided, it finds all submissions that have been modified since the start date)

Reviewer: ```suggestion
    - `last_modified_range`: (dict) a start and end date for the last modified date range. If either is not provided, the search is unbounded in that direction (e.g. if the end date is not provided, it finds all submissions that have been modified since the start date)
```
